ops/terraform/env/mgmt-test/stateless/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = "~> 0.12"
+}
+
+provider "aws" {
+  version = "~> 2.25"
+  region = "us-east-1"
+}
+
+module "stateless" {
+  source = "../../../modules/mgmt_stateless"
+
+  env_config = {
+    env               = "mgmt-test"
+    tags              = {application="bfd", business="oeda", stack="mgmt", Environment="mgmt-test"}
+  }  
+
+  jenkins_ami            = var.jenkins_ami
+  jenkins_key_name       = var.jenkins_key_name
+  jenkins_instance_size  = var.jenkins_instance_size
+  azs                    = var.azs
```

Comment:
In the current version of stateless/main.tf there is a variable for vpn_security_group_id, was this intentionally left out?

Reply:
This was removed, we now use a 'data' call to gather this vpn security group via the following code in the mgm_stateless module: 

```data "aws_security_group" "vpn" {
  filter {
    name        = "tag:Name"
    values      = ["bfd-mgmt-vpn-private"]
  }
}```